mdcat.1.adoc

```diff
@@ -102,6 +102,7 @@ If run as `mdless` or if `--paginate` is given and the pager fails to start mdca
 == Environment
 
 TERM::
+    If this variable is `wezterm`, mdcat assumes that the terminal is WezTerm.
```

Comment:
> If TERM_PROGRAM doesn't indicate wezterm (see below) and this variable is …"?

But TERM_PROGRAM can be anything as long as TERM is wezterm.

> So it's clear that you don't have to set both variables?

Both variables will be set as long as you are not in WSL or sshing into another machine.

Reply:
In the current implementation mdcat will only look at `TERM` if `TERM_PROGRAM` doesn't already indicate wezterm.

And in my experience and according to its own documentation wezterm sets TERM to `xterm-256color` in order to avoid extra terminfo files, see e.g. the [FAQ](https://wezfurlong.org/wezterm/faq.html#how-to-troubleshoot-keys-that-dont-work-or-produce-weird-characters) (emphasis mime):

> Interactive Unix programs generally depend upon the TERM environment variable being set appropriately. **wezterm sets this to xterm-256color by default**, because wezterm aims to be compatible with with the settings defined by that terminfo entry. Setting TERM to something else can change the byte sequences that interactive applications expect to see for some keys, effectively disabling those keys.

So as far as I can see TERM cannot identify wezterm sufficiently in its default configuration which is why I prefer for mdcat to look at TERM_PROGRAM first and only fall back to TERM in case TERM_PROGRAM isn't set (e.g. over SSH).  And to avoid any misunderstandings I'd like to have this documented, i.e. that mdcat only looks at TERM for wezterm if TERM_PROGRAM didn't yield a good result.

Reply:
> I'd like to avoid the impression that you need to set both TERM and TERM_PROGRAM to identify wezterm

No idea what to write. Please just commit what you want to see. I don't have time to bikesheed over this documentation part. I might as well just cherry-pick my first commit in nixpkgs.

Reply:
```suggestion
    If this variable is `wezterm`, mdcat assumes that the terminal is WezTerm. By default WezTerm does not set TERM to wezterm but mdcat can still detect it trough TERM_PROGRAM.
```

Reply:
Nor is the opposite written anywhere, so I'd like to have the facts spelled out explicitly.

Whether we can make progress here depends on you entirely.  I hold not stake here; you're asking me to merge a change on your behalf, not on mine, so I do not consider it my responsibility to amend these changes.  I'm merely stating my terms, and it's my way or the high way.

Please excuse me for being blunt, but I think you remember what you wrote before deleting it, and while I do give you credit for deleting it, I have read it nonetheless (remember, Github sends mail notifications for _every_ comment), so you'll understand that I'm less positively inclined towards this conversation now.

Commit your suggestion and I'll merge this pull request, and then you can go back to cherry-picking your nixpkg commits.